api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -74,6 +95,38 @@ public FireCloudServiceImpl(Provider<WorkbenchConfig> configProvider,
     this.workspacesApiProvider = workspacesApiProvider;
     this.statusApiProvider = statusApiProvider;
     this.retryHandler = retryHandler;
+    this.serviceAccounts = serviceAccounts;
+    this.firecloudCredentialProvider = firecloudCredentialProvider;
```

Comment:
nit: I'd probably include admin somewhere in this variable name; could potentially be misinterpreted as is